.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] Please add your name here instead of Author

![image](https://user-images.githubusercontent.com/74744787/164072195-88521eaa-ca00-43b0-8447-d81e44e71b0e.png)

Reply:
Great!